packaging/cmake/Config.cmake.in

```diff
@@ -0,0 +1,20 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+if(@WITH_ZLIB@)
```

Comment:
`SASL` added as dependency automatically here: https://github.com/edenhill/librdkafka/blob/f904298e774aae4776d25f074a071048b51dd53c/src/CMakeLists.txt#L100

You can verify it by looking at installed configs:

```
> grep sasl _install/lib/cmake/RdKafka/RdKafkaTargets.cmake
  INTERFACE_LINK_LIBRARIES "ZLIB::ZLIB;OpenSSL::SSL;OpenSSL::Crypto;Threads::Threads;sasl2"
```

It means '-lsasl2' will be added. If pkg-config return another name like just 'sasl', then it will be '...;sasl', etc. We don't have to do any extra steps as far as I understand.